src/state/airdrop/hooks.ts

```diff
@@ -75,3 +77,27 @@ export function useClaimCallback(
 
   return { claimCallback }
 }
+
+export interface Question {
+  id: number
+  title: string
+  content: string
+}
+export function useGetAirdropQuestions() {
+
+  return useQuery('getNews', async () => {
+    const response = await axios.get(`https://p7gm7mqi.directus.app/admin/content/kb/1`, {
```

Comment:
lets define this url somewhere on top of constant file `src/constants/index.ts`

Reply:
I completely deleted the function since it should not have appeared in this PR. sorry about that (coming in the next PR which will be dedicated to the airdrop only)